content/articles/why-kubernetes-is-the-best-technology-for-running-cloud-native-database/index.md

```diff
@@ -0,0 +1,69 @@
+﻿### Introduction
+For a long time, every organization’s dream is to migrate its workload to the cloud. However, the statistics from most IT organizations still show that there is still much work to be done.
+
+Although databases in the cloud have been around for a long time, the challenges with persisting and moving the cloud data have led to slow cloud adoption.
+
+For this reason, the demand for data infrastructure that takes full advantage of cloud computing has been on the rise.
+
+A [cloud-native database](https://www.networkcomputing.com/cloud-infrastructure/what-are-cloud-native-databases-and-why-should-you-use-them) has to be scalable, flexible, resilient, observable, and supports automation. A good example will be the [K8ssandra project](https://k8ssandra.io/get-started/), which bundles [Apache Cassandra](https://cassandra.apache.org/) and its supporting tools into production-ready Kubernetes deployment.
+
+A simple confusion arises if it is a must that any database running on Kubernetes be considered cloud-native. [Kubernetes](https://kubernetes.io/) was initially developed for [stateless workloads](https://dzone.com/articles/kubernetes-and-running-stateful-workloads), but with recent developments such as persistent volumes and stateful sets, it is now possible to run [stateful workloads](https://dzone.com/articles/kubernetes-and-running-stateful-workloads).
+
+The longtime DevOps practitioners who were at first skeptical about running databases on Kubernetes have now started adopting it, and best practices are emerging.
+
+For the databases to be more cloud-native as they should be, there is a great need to embrace what Kubernetes brings to the table.
+
+A truly cloud-native approach has to be followed, meaning adopting vital elements of the Kubernetes design paradigm. For example, a database that can work effectively on Kubernetes can be considered cloud-native.
+
+### Kubernetes Design Principles
```

Comment:
Connect these principles to the main objective of the article. For example, you can provide the following statement, 'Lets look at the design principles of Kubernetes to understand why it is the best technology for running cloud native databases.'